ftml/test/anchor2-alias-special.json

```diff
@@ -0,0 +1,43 @@
+{
+    "input": "[[*anchor_ href=\"http://example.com\" style=\"color: green;\"]]My link[[/anchor]]",
+    "tree": {
+        "elements": [
+            {
+                "element": "container",
+                "data": {
+                    "type": "paragraph",
+                    "elements": [
+                        {
+                            "element": "anchor",
+                            "data": {
+                                "attributes": {
+                                    "href": "http://example.com",
+                                    "style": "color: green;"
+                                },
+                                "target": "new-tab",
```

Comment:
(Out of scope for this PR) Should this be `_blank` or is that implementation left to the API consumer?

Reply:
`AnchorTarget` is defined at https://github.com/scpwiki/wikijump/blob/develop/ftml/src/tree/anchor.rs#L24-L45. The HTML-equivalent fields are listed in the documentation there, however for the purposes of an AST we should be descriptive and avoid HTML-specifics when possible.

This field in the JSON is a straight mapping to the enum, it's just saying `AnchorTarget::NewTab`.